tests/run-test-suite/src/libyaml-emitter.c

```diff
@@ -0,0 +1,253 @@
+#include <yaml.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include <assert.h>
+
+bool get_line(FILE *input, char *line);
+char *get_anchor(char sigil, char *line, char *anchor);
+char *get_tag(char *line, char *tag);
+void get_value(char *line, char *value, int *style);
+
+int main(int argc, char *argv[]) {
```

Comment:
Starting to think we need some kind of style checker/enforcer on this project for C code and such. This style differs vastly from the rest of the project (which is probably fine, but I'd love to keep it consistent to avoid butting heads with future contributors over matching the style of the file they're working on).